### PR TITLE
[TASK] Add a README for the version handling package

### PR DESCRIPTION
**Pull request by linawolf**
This should also trigger the initial split deployment

### CODE REVIEW

**Review by garvinhicking — APPROVED** [review]

File: packages/typo3-version-handling/README.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+==========================================================
+Version handling for PHP based rendering of docs.typo3.org
+==========================================================
+
+This package contains a PHP enum that can be used to determine allowed TYPO3
+versions for documentation interlinking. It also allows a matching between
+relative versions, for example `stable` and absolute minor versions, for
```
Comment (brotkrueml) [inline packages/typo3-version-handling/README.rst:8]: i had to read it three times to understand the sencence. I would avoid "relative" and "absolute", maybe better:

descriptive versions like stable

version numbers like 12.4

File: packages/typo3-version-handling/README.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+example `12.4`.
+
+Additionally it contains an enum with a list of supported standard inventories
```
Comment (brotkrueml) [inline packages/typo3-version-handling/README.rst:11]:
```suggestion
Additionally, it contains an enum with a list of supported standard inventories
```

File: packages/typo3-version-handling/README.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+of official manuals. These are used for interlink mapping.
```
Comment (brotkrueml) [inline packages/typo3-version-handling/README.rst:12]:
```suggestion
of official references, tutorials and guides. These are used for interlink mapping.
```

First, I thought, also sysexts are meant. Bu currently they aren't in it.

File: packages/typo3-version-handling/README.rst (hunk 1 above)
Comment (linawolf) [inline packages/typo3-version-handling/README.rst:8]:
```suggestion
descriptive versions like  `stable` and absolute minor versions, for
```

**Review by garvinhicking — APPROVED** [review]

File: packages/typo3-version-handling/README.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
+==========================================================
+Version handling for PHP based rendering of docs.typo3.org
+==========================================================
+
+This package contains a PHP enum that can be used to determine allowed TYPO3
+versions for documentation interlinking. It also allows a matching between
+descriptive versions like  `stable` and absolute minor versions, for
```
Comment (brotkrueml) [inline packages/typo3-version-handling/README.rst:8]: Hmm, I don't like the "absolute minor versions" - I just don't understand this term (but I know what you mean).

**Review by brotkrueml — APPROVED** [review]

File: packages/typo3-version-handling/README.rst (hunk 4 above)
Comment (linawolf) [reply packages/typo3-version-handling/README.rst:8]: can you do a follow up?

**Pull request merged**